apps/css-workshop/src/pages/badge.astro

```diff
@@ -1,5 +1,6 @@
 ---
 import Layout from './_layout.astro';
+import IuiBadge from '../components/Badge.astro';
```

Comment:
This is the error I get when using `Badge` inside `badge.astro`:

![image](https://github.com/iTwin/iTwinUI/assets/9084735/b88b305a-84ef-4b55-9556-8ee3c2b76334)

I don't like `IuiBadge`, it feels kinda ugly/long. Open to suggestions.

Reply:
maybe `AstroBadge` or `BadgeAst` ?

Reply:
Astro is just the framework we are using today. The contents of the component have nothing to do with Astro.

Maybe `Badge_` could work?

![image](https://github.com/iTwin/iTwinUI/assets/9084735/4ce60db8-de9a-43fa-80b9-5bfcba8a82ac)
![image](https://github.com/iTwin/iTwinUI/assets/9084735/29be5f21-2d26-451d-8f97-3cbebdec9772)